daemon/indexing/lib/search.js

```diff
@@ -113,16 +115,74 @@ class Listing {
         id: hit._id,
         name: hit._source.name,
         description: hit._source.description,
-        price: hit._source.price,
+        priceEth: hit._source.priceEth,
       }
       listings.push(listing)
     })
     return listings
   }
 }
 
+class Offer {
+  /*
+   * Indexes an Offerr
+   * @params {object} offer - JSON offer data from origin.js
+   * @throws Throws an error if indexing operation failed. 
+   */
+  static async index(offer, listing){
+    const resp = await client.index({
+      index: OFFER_INDEX,
+      type: OFFER_TYPE,
+      id: offer.id,
+      body: {
+        id: offer.id,
+        listingId: offer.listingId,
+        buyer: offer.buyer,
+        seller: listing.seller,
+        affiliate: offer.affiliate,
+        priceEth: offer.priceEth,
+        status: offer.status
+      }
+    })
+  }
+
+  static async get(id) {
+    const resp = await client.get({id: id, index: OFFER_INDEX, type: OFFER_TYPE})
+    if(!resp.found){
+      throw "Offer not found"
```

Comment:
nit: I'm a JS noob so asking :) Should we prefer throwing an Error rather than a String as a general convention in our stack ?

Reply:
Will update